### ISSUE TITLE
Include vcs.h - unable to load

### ISSUE THREAD

**Issue by MuellerMuellerNM** [zeh/vscode-dasm#1]
Created a new test.asm with the following code

processor 6502
include “vcs.h“

i get the following error [dasm] unable to losd vcs.h

What could that be?

**Comment by vbauer** [comment]
@zeh Hello! I have the same problem. Is it some local problem with Visual Studio code or bug in the plugin?

**Comment by zeh** [comment]
Hey @vbauer, @MuellerMuellerNM,

Interesting, for some reason the built-in vcs.h is not being used when it encounters that.

I'll take a look at it, but two solutions I'd like you to try:

1. Do `include "vcs.h"`, with quotes.
2. Try making sure your own `vcs.h` (if supplied) is in the same folder as the source file (ie pong.asm)

**Comment by vbauer** [comment]
Hello @zeh! Thank you for suggestion, I've already tried to change vcs.h to "vcs.h" and even to 'vcs.h', but it does not help

Btw, I've found some strange error in output tab, maybe it helps:
```
[pm] NEW CURRENT URI is  file:///Users/vbauer/Documents/workspace/atari2600/Punisher-Pong/pong.asm
[pm] ...ALSO SETTING new project
---
 ALL tabs:  [ 'file:///Users/vbauer/Documents/workspace/atari2600/Punisher-Pong/pong.asm' ]
 Projects:  1
 CURRENT project:
  1 files: 
    0: file:///Users/vbauer/Documents/workspace/atari2600/Punisher-Pong/pong.asm
      has document = false
      has content = false, length = -1
      is dirty = false
      version = -1
      includes = 0
---
/Users/vbauer/.vscode/extensions/zehfernando.vscode-dasm-3.2.0/out/server/node_modules/dasm/lib/index.js:76
                        address = parseNumber(rawLine.match(addressFind)[1]);
                                                                        ^

TypeError: Cannot read property '1' of null
    at /Users/vbauer/.vscode/extensions/zehfernando.vscode-dasm-3.2.0/out/server/node_modules/dasm/lib/index.js:76:73
    at Array.forEach (native)
    at parseList (/Users/vbauer/.vscode/extensions/zehfernando.vscode-dasm-3.2.0/out/server/node_modules/dasm/lib/index.js:35:14)
    at Object.default_1 [as default] (/Users/vbauer/.vscode/extensions/zehfernando.vscode-dasm-3.2.0/out/server/node_modules/dasm/lib/index.js:437:27)
    at Assembler.assemble (/Users/vbauer/.vscode/extensions/zehfernando.vscode-dasm-3.2.0/out/server/providers/Assembler.js:7:38)
    at Project.assemble (/Users/vbauer/.vscode/extensions/zehfernando.vscode-dasm-3.2.0/out/server/project/Project.js:71:45)
    at Project._files.onChanged.add (/Users/vbauer/.vscode/extensions/zehfernando.vscode-dasm-3.2.0/out/server/project/Project.js:13:18)
    at /Users/vbauer/.vscode/extensions/zehfernando.vscode-dasm-3.2.0/out/server/node_modules/simplesignal/dist/SimpleSignal.commonjs.js:43:18
    at Array.forEach (native)
    at SimpleSignal.dispatch (/Users/vbauer/.vscode/extensions/zehfernando.vscode-dasm-3.2.0/out/server/node_modules/simplesignal/dist/SimpleSignal.commonjs.js:42:28)
[Error - 00:24:04] Connection to server got closed. Server will not be restarted.
```

vcs.h in the same folder as it was shown on screenshot above:
```bash
➜  Punisher-Pong git:(master) ✗ ls
README   pong.asm pong.bin vcs.h
```
// The following project: https://github.com/avanco/Punisher-Pong

**PS:**
* VS Code: Version 1.25.1 (1.25.1) 1dfc5e557209371715f655691b1235b6b26a06be 2018-07-11T15:27:07.646Z
* OS: macOS High Sierra, version 10.13.6 (17G65)

**Comment by zeh** [comment]
Nice, thanks @vbauer! Yeah the problem seems to be on the project parser. Strange. I'll take a look when I can (it might be a few days). Having the project's GH helps, thanks!

In the meantime, sorry about the bug.

**Comment by zeh** [comment]
Hey @vbauer. I just realized this is likely a bug that _was_ fixed [on a previous commit](https://github.com/zeh/vscode-dasm/commit/8569fbbc453fd83d04d8892a7692bb59c7c7b04a)... but was only rolled into the "debugger" branch which is not released yet. :(

I've just released version 4.0.0 now integrating this fix and others. The caveat is that I brought a lot of changes and refactorings from the debugger branch. While it seems to be working, I might have missed some bug.

If it's not too late, can I ask you to try it again? I'll only be able to test it on a macOS system later this week.

**Comment by zeh** [comment]
Ok, I've verified version 4.0.0 is working on macOS 10.13.6 with the Punisher-Pong repository.

Would appreciate more testing, but included to close this issue soon!

**Comment by zeh** [comment]
Fixed in 4.0.0.

**Comment by vbauer** [comment]
I'm really sorry for my too late response.. I've just checked it and everything works well on my macOs (High Sierra 10.13.6) with VS Code 1.27.2

Thank you very much!
